src/redux/rocket/rocket.js

```diff
@@ -0,0 +1,61 @@
+const STORE_ROCKET_DATA = 'space-travelers-hub/rocket/STORE_ROCKET_DATA';
+const RESERVE_ROCKET_TICKET = 'space-travelers-hub/rocket/RESERVER_ROCKET_TICKET';
+const CANCEL_ROCKET_TICKET = 'space-travelers-hub/rocket/CANCEL_ROCKET_TICKET';
+
+const defaultState = [];
+
+const rocketReducer = (state = defaultState, action) => {
+  switch (action.type) {
+    case STORE_ROCKET_DATA:
+      return action.payload.map((item) => {
+        const {
+          rocket_id: id,
+          rocket_type: type,
+          rocket_name: name,
+          flickr_images: flickrImages,
+          description,
+          wikipedia,
+        } = item;
+        return {
+          id, type, name, flickrImages, description, wikipedia,
+        };
+      });
+    case RESERVE_ROCKET_TICKET:
+      return state.map((rocket) => {
+        if (rocket.id !== action.id) return rocket;
+        return { ...rocket, reserved: true };
+      });
```

Comment:
- [ ] It looks like you're sending the id for the rocket, and that's good, now it would be great if you save on redux store the next information:
- id
- rocket_name
- description
- flickr_images (the first one)
I check this reducer switch option and it looks like it only saves the id on the redux store, please work on this. 🐶